code/campaignclassic/src/main/java/com/adobe/marketing/mobile/AEPPushTemplate.java

```diff
@@ -401,7 +401,7 @@ long getRemindLaterTimestamp() {
         return ticker;
     }
 
-    boolean getNotificationAutoCancel() {
+    boolean getNotificationStickySetting() {
```

Comment:
nit: For booleans, readability is easier when it is worded "is<Boolean>" - `isSticky()`.